dll/win32/kernel32/client/proc.c

```diff
@@ -1588,13 +1588,18 @@ FatalAppExitW(IN UINT uAction,
 {
     UNICODE_STRING UnicodeString;
     ULONG Response;
+#if DBG
     NTSTATUS Status;
+#endif
 
     /* Setup the string to print out */
     RtlInitUnicodeString(&UnicodeString, lpMessageText);
 
     /* Display the hard error no matter what */
-    Status = NtRaiseHardError(STATUS_FATAL_APP_EXIT | HARDERROR_OVERRIDE_ERRORMODE,
+#if DBG
+    Status =
```

Comment:
`UNREFERENCED_LOCAL_VARIABLE(Status)` seems nicer.

Reply:
or DBG_UNREFERENCED_LOCAL_VARIABLE?

Reply:
DBG_UNREFERENCED_LOCAL_VARIABLE means the code referencing the variable is not implemented and it will be fixed later - it has no relation to the DBG macro.

Reply:
> `UNREFERENCED_LOCAL_VARIABLE(Status)` seems nicer.

I feel that would be an abuse of that macro and I would need to duplicate the `NtRaiseHardError()` call,
so I think my proposal is clearer.

Reply:
Duplicate? Not sure I follow. Here's what I'm thinking:
```
NTSTATUS Status;
Status = NtRaiseHardError(...)
#if DBG
if (!NT_SUCCESS(Status)) { ... }
#else
UNREFERENCED_LOCAL_VARIABLE(Status);
#endif
```

Reply:
Yeah, I guessed that and I feel it is an abuse of that macro. Anyway.
Changed. As you prefer.